02-csv-vector-raster-plotting.Rmd

```diff
@@ -24,29 +26,27 @@ comments: false
 #About
 
 This lesson will review how to import spatial points stored in a .csv file into
-R as a spatial object - a `SpatialPointsDataFrame`. We will also learn how to
-reproject data imported in shapefile format into another projection, create a 
-shapefile from a spatial object in R, and plot raster and vector data as overlays
-in the same plot. 
+`R` as a spatial object - a `SpatialPointsDataFrame`. We will also learn how to
+reproject data in a shapefile format into another projection, create a 
```

Comment:
subtle but i thinkt he previous text was more accurate. they actually aren't reprojecting a shapefile. they are importing a shapefile into a spatial object, reprojecting the spatial object and THEN exporting a shapefile.